.circleci/config.yml

```diff
@@ -140,8 +140,10 @@ jobs:
     executor: test-executor
     steps:
       - browser-tools/install-browser-tools:
-          chrome-version: 116.0.5845.96 # TODO: remove when chromedriver downloads are fixed
```

Comment:
question - just checking that this [Issue resolution for the orb](https://github.com/CircleCI-Public/browser-tools-orb/issues/95#issuecomment-1718306019), confirms that ChromeDriver's can now be installed? 

Just asking as there is a recent [open issue](https://github.com/CircleCI-Public/browser-tools-orb/issues/99), where chrome failed to start. Not sure if we'll encounter the same thing?

Reply:
the original issue and the linked one both refer to issues with chromedriver being installed, in this PR I have removed the installation of chromedriver and the tests are all working as expected.